product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.1-converting-soap-to-json/1.1.1-soap-to-json-using-payloadfactory-mediator/src/test/java/org/wso2/carbon/esb/endpoint/test/ScenarioSampleTest.java

```diff
@@ -0,0 +1,178 @@
+    private final String carFileName = "SOAPToJSONCarbonApplication_1.0.0";
+    int timeout = 20;
+    RequestConfig requestConfig = RequestConfig.custom()
+                                               .setConnectTimeout(timeout * 100)
+                                               .setConnectionRequestTimeout(timeout * 1000)
+                                               .setSocketTimeout(timeout * 1000).build();
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        infraProperties = getDeploymentProperties();
+        String backendURL = infraProperties.getProperty(CARBON_SERVER_URL) + "/";
+        setKeyStoreProperties();
+        AuthenticatorClient authenticatorClient = new AuthenticatorClient(backendURL);
+        String sessionCookie = authenticatorClient.login("admin", "admin", getServerHost());
+        log.info("The Backend service URL : " + backendURL + ". session cookie: " + sessionCookie);
+
+        carbonAppUploaderClient = new CarbonAppUploaderClient(backendURL, sessionCookie);
+        DataHandler dh = new DataHandler(new FileDataSource(new File(resourceLocation + File.separator + "artifacts" +
+                                                                     File.separator + carFileName + ".car")));
+        carbonAppUploaderClient.uploadCarbonAppArtifact(carFileName + ".car", dh);
+        applicationAdminClient = new ApplicationAdminClient(backendURL, sessionCookie);
+        // Wait for Capp to sync
+        Thread.sleep(60000);
+        Assert.assertTrue(isCarFileDeployed(carFileName), "Car file deployment failed");
+    }
+
+    @Test(description = "1.1.1.1", enabled = true)
+    public void testMessageTransformation() throws Exception {
+        // Invoke the service and invoke
+        String restURL = infraProperties.getProperty(ESB_HTTP_URL) + "/city/lookup/60601";
+        log.info("The API Endpoint : " + restURL);
+        HttpGet httpGet = new HttpGet(restURL);
+        Thread.sleep(1000);
+        try (CloseableHttpClient httpClient = HttpClientBuilder.create().setDefaultRequestConfig(requestConfig).build()) {
+            try (CloseableHttpResponse response = httpClient.execute(httpGet)) {
+                log.info("The response status : " + response.getStatusLine());
+                String responseString = "{\n" +
+                                        "  \"LookupCityResult\": {\n" +
+                                        "    \"City\": \"Chicago\",\n" +
+                                        "    \"State\": \"IL\",\n" +
+                                        "    \"Zip\": 60601\n" +
+                                        "  }\n" +
+                                        "}";
+                Assert.assertEquals(response.getStatusLine().getStatusCode(), 200);
+                Assert.assertEquals(IOUtils.toString(response.getEntity().getContent()), responseString);
+            }
+        } catch (IOException e) {
```

Comment:
Why do we have an additional catch block here?

Reply:
This file has been removed by PR [https://github.com/wso2/product-ei/pull/2977](https://github.com/wso2/product-ei/pull/2977)